kernel/data-pipeline/dialect/mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/client/MySQLClient.java

```diff
@@ -270,6 +280,10 @@ private final class MySQLBinlogEventHandler extends ChannelInboundHandlerAdapter
         
         private AbstractBinlogEvent lastBinlogEvent;
```

Comment:
Could `lastBinlogEvent` be final and simplify constructor?

Reply:
OK.

1, Could we use `@AllArgsConstructor` to replace constructor?

2, Could we add `volatile` for `lastBinlogEvent`?